mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule PropCheck.Mixfile do
   def project do
     [app: :propcheck,
      version: "1.2.3-dev",
-     elixir: "~> 1.5",
+     elixir: ">= 1.7.0",
```

Comment:
This would allow Elixir 2.x. We should not be so optimistic :-) Please change to `~> 1.7`, which requires a 1.x version of Elixir.